x-pack/plugins/endpoint/public/applications/endpoint/store/hosts/reducer.ts

```diff
@@ -40,19 +40,19 @@ export const managementListReducer: Reducer<ManagementListState, AppAction> = (
       pageIndex,
       loading: false,
     };
-  } else if (action.type === 'serverReturnedManagementDetails') {
+  } else if (action.type === 'serverReturnedHostDetails') {
     return {
       ...state,
       details: action.payload,
     };
-  } else if (action.type === 'serverFailedToReturnManagementDetails') {
+  } else if (action.type === 'serverFailedToReturnHostDetails') {
     return {
       ...state,
       detailsError: action.payload,
     };
-  } else if (action.type === 'userExitedManagementList') {
+  } else if (action.type === 'userExitedHostList') {
```

Comment:
is this action dispatched anywhere? what would it mean?

Reply:
Sorry I thought I got rid of that action! It's supposed to be gone.